gui/app/directives/chat/chat-user-category.js

```diff
@@ -15,7 +15,7 @@
                 <div style="font-size: 12px; opacity: 0.6;">{{$ctrl.category}}</div>
                 <div
                     class="chat-user-wrapper"
-                    ng-repeat="user in cms.chatUsers | chatUserRole:$ctrl.roleKey | orderBy:'username':true | orderBy:'active':true as filtered track by user.id"
+                    ng-repeat="user in cms.getFilteredChatUserList() | chatUserRole:$ctrl.roleKey | orderBy:'username':true | orderBy:'active':true as filtered track by user.id"
```

Comment:
Note the viewer list will always run through this new filter function instead of pulling directly from our main chatUsers array.